### PR TITLE
[Hub Generated] Review request for Microsoft.Marketplace to add version stable/2020-01-01

### PR DESCRIPTION
**Pull request by MiriBerezin**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/MiriBerezin/azure-rest-api-specs/Marketplace-miberezi-2020...master/).

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “**WaitForARMFeedback**” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [x] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by MiriBerezin** [comment]
Breaking changes - I'm fixing the Live Validation Correctness Error Details. 
This includes  new properties on the Privatestore and offer responses

**Comment by lirenhe** [comment]
@MiriBerezin, as the PR contains breaking changes, you need to contact @azbreakchangereview for approval. Please mention to them this is to fix the Swagger Correctness issue for S360.

**Comment by MiriBerezin** [comment]
@lirenhe , @JeffreyRichter 

As discussed in mails, those changes were already done in May with the same API version while the (previous) team was still developing the service. 

So even if we change the API version that only means we will have two api-versions that are exactly the same. 

Since those changes were already done we are asking for an exception on that.

We are on public preview since September and the breaking changes were done in May. 

Please help me to finish this issue and let me know what are the next steps
Thanks!

**Review by JeffreyRichter — APPROVED** [review]

File: specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-01-01/Marketplace.json [hunk 1]
```diff
@@ -575,6 +581,37 @@
...
+        "hasCommercialAssociation": {
+          "description": "Indicating whether private store has association with Commercial's Billing Account (through billing account's customer tag retrieved from GSM for a subscription",
+          "type": "boolean"
+        },
+        "hasMultiTenantAssociation": {
+          "description": "Indicating whether private store has association with multiple tenants (through tenant's tag retrieved from AAD",
+          "type": "boolean"
+        },
+        "isGov": {
+          "description": "",
```
Comment (lirenhe) [inline specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-01-01/Marketplace.json:613]: will you provide the description for this property?

**Review by lirenhe — APPROVED** [review]

**Pull request merged**